src/services/redisSessionStorage.ts

```diff
@@ -5,16 +5,20 @@
 import { Either, isLeft, left, right } from "fp-ts/lib/Either";
 import { ReadableReporter } from "italia-ts-commons/lib/reporters";
 import * as redis from "redis";
-import { isNumber } from "util";
+import { isNumber, promisify } from "util";
 import { SessionToken, WalletToken } from "../types/token";
 import { User } from "../types/user";
 import { log } from "../utils/logger";
 import { ISessionStorage } from "./ISessionStorage";
 
-const sessionKeyPrefix = "SESSION-";
-const walletKeyPrefix = "WALLET-";
+const sessionKeyPrefix = `SESSION-`;
+const walletKeyPrefix = `WALLET-`;
+const userKeyPrefix = `USER-`;
 
 export default class RedisSessionStorage implements ISessionStorage {
+  private sessionNotFoundMessage = "Session not found";
```

Comment:
this is a const, can stay outside the class

Reply:
fixed in 155b65b